### PR TITLE
videojs-standard Control Bar

### PR DESCRIPTION
**Pull request by misteroneill**
## Description
Makes `src/js/control-bar/` pass videojs-standard.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by gkatsev** [comment]
Linting errors :P

**Comment by gkatsev** [comment]
I think this PR needs to be re-created against the other branch?

File: src/js/control-bar/control-bar.js [hunk 1]
```diff
@@ -4,24 +4,24 @@
...
-import VolumeMenuButton from './volume-menu-button.js';
-import MuteToggle from './mute-toggle.js';
-import ChaptersButton from './text-track-controls/chapters-button.js';
-import DescriptionsButton from './text-track-controls/descriptions-button.js';
-import SubtitlesButton from './text-track-controls/subtitles-button.js';
-import CaptionsButton from './text-track-controls/captions-button.js';
-import AudioTrackButton from './audio-track-controls/audio-track-button.js';
-import PlaybackRateMenuButton from './playback-rate-menu/playback-rate-menu-button.js';
-import CustomControlSpacer from './spacer-controls/custom-control-spacer.js';
+import './play-toggle.js';
```
Comment (gkatsev) [inline src/js/control-bar/control-bar.js]: I guess we should add the same message here as in `player.js`?

File: src/js/control-bar/mute-toggle.js [hunk 2]
```diff
@@ -52,7 +53,7 @@ class MuteToggle extends Button {
    * @method handleClick
    */
   handleClick() {
-    this.player_.muted( this.player_.muted() ? false : true );
+    this.player_.muted(this.player_.muted() ? false : true);
```
Comment (gkatsev) [inline src/js/control-bar/mute-toggle.js]: I really want this changed to `this.player_.muted(!this.player_.muted())`.

Comment (gkatsev) [inline src/js/control-bar/mute-toggle.js]: But for another PR...

File: src/js/control-bar/text-track-controls/chapters-button.js [hunk 3]
```diff
@@ -73,59 +69,62 @@ class ChaptersButton extends TextTrackButton {
    * @method createMenu
    */
   createMenu() {
-    let tracks = this.player_.textTracks() || [];
+    const tracks = this.player_.textTracks() || [];
     let chaptersTrack;
-    let items = this.items || [];
+    const items = this.items || [];
```
Comment (gkatsev) [inline src/js/control-bar/text-track-controls/chapters-button.js]: items needs to be `let` here.

**Comment by gkatsev** [comment]
Commented on the commit. The linter complains about `items` being const, it needs to be `let`, at least for this PR because otherwise you'll need to introduce functional changes.
LGTM otherwise.

File: src/js/control-bar/text-track-controls/chapters-button.js (hunk 3 above)
Comment (misteroneill) [inline src/js/control-bar/text-track-controls/chapters-button.js]: Whoops! 😄

File: src/js/control-bar/mute-toggle.js (hunk 2 above)
Comment (misteroneill) [inline src/js/control-bar/mute-toggle.js]: Same. I almost did. The temptation was great, but I resisted!

**Pull request merged**